src/components/itemHelper.js

```diff
@@ -300,6 +327,7 @@ export default {
     canIdentify: canIdentify,
     canEdit: canEdit,
     canEditImages: canEditImages,
+    canEditSubtitles: canEditSubtitles,
```

Comment:
I know the rest of this export is written this way, but it is just a legacy pattern. We can just do this instead:

```suggestion
    canEditSubtitles,
```